src/AndersonAcceleration.f90

```diff
@@ -85,109 +85,56 @@ MODULE AndersonAccelerationTypes
 !-------------------------------------------------------------------------------
 !> @brief Initializes the Anderson Acceleration Type with a parameter list
 !> @param pList the parameter list
-!>
 !> @param solver The anderson acceleration solver to act on
 !> @param MPIEnv The MPI environment description
-!> @param Params The aprameterlist
-!>
-!> This routine initializes the data spaces for the NOX Anderson Acceleration
+!> @param Params The anderson options parameter list
 !>
 SUBROUTINE init_AndersonAccelerationType(solver,MPIEnv,Params)
```

Comment:
Pass in a `FutilityComputingEnvironmentType` here instead of an mpi object.  Then delete `eAndersonAccelerationType` and just use the computing environment exceptional handler.

Reply:
Good idea!